client.go

```diff
@@ -343,7 +336,14 @@ func (c *Client) doRequest(ctx context.Context, method, uri string, body interfa
 	}
 
 	req.Header.Set("Content-Type", "application/json")
-	req.Header.Set("User-Agent", c.UserAgent)
+	req.Header.Set(
+		"User-Agent",
+		strings.TrimSpace(fmt.Sprintf(
+			"go-client/%s %s",
```

Comment:
```suggestion
			"go-client/%s/%s",
```
I think, it shouldn't have spaces.

Reply:
It should have spaces as per the RFC. The final User-Agent is going to look like: `go-client/1.0.0 terraform-provider-aiven/1.0.0`.